internal/storetest/storedata.go

```diff
@@ -101,17 +102,57 @@ func (storeData *StoreData) LoadModel(basePath string) (authorizationmodel.Model
 	return format, nil
 }
 
+//nolint:gocognit,cyclop
```

Comment:
@aaguiarz 
I am wondering if we do a tuple duplicate check like such
the cyclomatic complexity increases failing these linters
What is your suggestion?